backend/test/session.test.js

```diff
@@ -8,10 +8,14 @@ describe('sessions', () => {
       .set('Content-Type', 'application/json')
       .send({ name: 'Lehel', password: 'asdf' })
       .expect('Content-Type', /json/)
-      .expect(200).toHaveProperty('token')
-      .end(function(err, res) {
-        if (err) return done(err);
-        done();
+      .expect(200)
+      .end((err, res) => {
+        if (err) {
+          return done(err);
+        }
+        expect(res.hasOwnProperty('token'))
```

Comment:
```suggestion
        expect(res).toHaveProperty('token');
```
it's almost the same, but you get to use the `toHaveProperty` now, because that expect is now from `jest`